src/main/java/org/testng/xml/XmlTest.java

```diff
@@ -582,7 +582,7 @@ public String toXml(String indent) {
   @Override
   public String toString() {
 //    return toXml("");
-    StringBuffer result = new StringBuffer("[Test: \"" + m_name + "\"")
+    StringBuilder result = new StringBuilder("[Test: \"" + m_name + "\"")
```

Comment:
Actually, I'm beginning to think twice about this. Favoring StringBuilder over StringBuffer seems like a best practice that was probably made obsolete by the introduction of biased locking in JDK6. There are no readability differences between the two, and I'm not trying to fix imaginary performance problems. I _would_ like to see `String#format` preferred over string concatenation or appending to buffers, but that's another story.